src/types/magnitude.rs

```diff
@@ -94,25 +95,6 @@ impl Magnitude {
         }
         digits
     }
-
-    fn calculate_u64_digits(value: u64) -> u64 {
```

Comment:
🗺️ This method was being used in a couple of different places, so I moved it to the module root for common access.